src/virtual-source-buffer.js

```diff
@@ -437,10 +437,6 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
     }
 
     if (this.appendInitSegment_) {
```

Comment:
Same for naming of the 'only appends init segment for first segment or on audio/media changes' test